apps/i18n/common/en_us.json

```diff
@@ -1126,6 +1126,7 @@
   "selectGoogleClassroom": "Select a Google Classroom",
   "selectCleverSection": "Select a Clever section",
   "selectSection": "Select Section",
+  "selectSectionLabel": "Select section:",
```

Comment:
Is there a reason this string has a lower case "section" while the one above is capitalized? Seems like we should be consistent across the site, but I don't know which casing we've settled on for labels like this.

Reply:
fair question! this appears to be what we show inside the dropdown when no section is selected:
<img width="203" alt="screen shot 2018-08-30 at 1 43 19 pm" src="https://user-images.githubusercontent.com/8001765/44878067-c4960400-ac5a-11e8-851e-c7da27d4b8c0.png">
so it's more of a placeholder, and less of a label, I guess? In any case I don't think it's especially important for these to appear identical since they serve different purposes.